src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/Appointment.json

```diff
@@ -0,0 +1,103 @@
+{
+    "resourceType": "Appointment",
+    "id": "example",
+    "text": {
+        "status": "generated",
+        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Brian MRI results discussion</div>"
+    },
+    "status": "booked",
+    "serviceCategory": [
+        {
+            "coding": [
+                {
+                    "system": "http://example.org/service-category",
+                    "code": "gp",
+                    "display": "General Practice"
+                }
+            ]
+        }
+    ],
+    "serviceType": [
+        {
+            "coding": [
+                {
+                    "code": "52",
+                    "display": "General Discussion"
+                }
+            ]
+        }
+    ],
+    "specialty": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "394814009",
+                    "display": "General practice"
+                }
+            ]
+        }
+    ],
+    "appointmentType": {
+        "coding": [
+            {
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0276",
+                "code": "FOLLOWUP",
+                "display": "A follow up visit from a previous appointment"
+            }
+        ]
+    },
+    "reasonReference": [
```

Comment:
[https://www.hl7.org/fhir/stu3/appointment.html](https://www.hl7.org/fhir/stu3/appointment.html) there is no `reasonReference` in STU3

Reply:
Updated!  Thanks for the catch.